proteus/mprans/GN_SW2DCV.cpp

```diff
@@ -5,30 +5,28 @@
 #include "GN_SW2DCV.h"
 
 #if defined(__GNUC__) && !defined(__clang__)
-    namespace workaround
-    {
-        inline void define_allocators()
-        {
-            std::allocator<int> a0;
-            std::allocator<double> a1;
-        }
-    }
+namespace workaround {
+inline void define_allocators() {
+  std::allocator<int> a0;
+  std::allocator<double> a1;
+}
+} // namespace workaround
```

Comment:
We don't have a mandated indentation style for C++ and probably won't adopt one. So if you want to use K&R style instead of Allman style, it's fine by me, but it's better not to touch code that you're not changing unless you have a good reason. Making it all consistent in files that you are going to take ownership of is good enough reason, but you might want to do it in a separate PR so reviewers can focus on the substantive changes.